torch/distributed/tensor/placement_types.py

```diff
@@ -698,11 +646,6 @@ def _partition_value(
         num_chunks = mesh.size(mesh_dim=mesh_dim)
         return tensor / num_chunks
 
-    def __eq__(self, other: object) -> bool:
-        if not isinstance(other, Partial):
-            return False
-        return self.reduce_op == other.reduce_op
-
     def __hash__(self) -> int:
```

Comment:
Should we rely on super method hashing here now?

Reply:
Why? I don't think there's an impetus for an algorithm change here.